packages/e2e-tests/plugins/interactive-blocks/directive-on-window/render.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * HTML for testing the directive `data-wp-on`.
+ *
+ * @package gutenberg-test-interactive-blocks
+ */
+
+gutenberg_enqueue_module( 'directive-on-window-view' );
+?>
+
+<div data-wp-interactive='{ "namespace": "directive-on-window" }'>
+	<div data-wp-on-window--resize="callbacks.resizeHandler">
```

Comment:
Same here.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/57931/commits/99e2ed82f985caff54a8a09745b91fca39241464